go.mod

```diff
@@ -1,65 +1,62 @@
 module github.com/BitBoxSwiss/bitbox-wallet-app
 
-go 1.20
+go 1.21
```

Comment:
This is the minimum version, so it can technically stay at 1.20, but it's a good idea to bump it to enforce the release build machines to update to this version.